### DISCUSSION TITLE
which directory is default data?

### DISCUSSION THREAD

**Discussion by epubreader** [apache/kvrocks#2459] [Q&A] [answered]
which directory is default data? /tmp/kvrocks or /var/lib/kvrocks

is this correct?
docker service create \
  --name kvrocks \
  --publish 6666:6666 \
  --mount type=bind,source=/home/owner/docker/kvrocks/kvrocks.conf,target=/kvrocks/conf/kvrocks.conf  \
  --mount type=bind,source=/home/owner/docker/kvrocks/data,target=/data \
  --mode global \
  --restart-condition on-failure \
  --with-registry-auth \
  apache/kvrocks:2.9.0

**Comment by git-hulk** [comment]
It will be inside `/var/lib/kvrocks`.

**Reply by epubreader** [reply]
Can I run docker by mounting the conf configuration file?
I tried it, but it didn't work

**Reply by git-hulk** [reply, marked as answer]
Did you mount the volume to `/var/lib/kvrocks`?
